src/main/java/io/github/debuggyteam/tablesaw/TableSawRecipes.java

```diff
@@ -1,24 +1,19 @@
 package io.github.debuggyteam.tablesaw;
 
-import java.util.ArrayDeque;
-import java.util.ArrayList;
-import java.util.Deque;
-import java.util.List;
-import java.util.Map;
-
 import com.google.common.collect.HashMultimap;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.Multimap;
-
 import io.github.debuggyteam.tablesaw.api.TableSawRecipe;
 import net.fabricmc.api.EnvType;
 import net.fabricmc.api.Environment;
 import net.minecraft.item.Item;
 import net.minecraft.item.ItemConvertible;
 import net.minecraft.item.ItemStack;
 
+import java.util.*;
```

Comment:
Star import intended?

Reply:
Likely happened due to import optimization.